shared/actions/tracker.js

```diff
@@ -621,6 +631,26 @@ function getTracking (username: string): Promise {
   })
 }
 
+function fillFolders (getState: () => {favorite: FavoriteState}, username: string): TrackerActionCreator {
+  return (dispatch, getState) => {
+    const root = getState().favorite.folders || {}
+    const pubIg = root.public.ignored || []
+    const pubTlf = root.public.tlfs || []
+    const privIg = root.private.ignored || []
+    const privTlf = root.private.tlfs || []
+
+    const tlfs = [].concat(pubIg, pubTlf, privIg, privTlf).filter(f => f.users.filter(u => u.username === username).length)
```

Comment:
Just asking if the list of folders were showing in the profile includes
ignored folders

On Friday, June 24, 2016, Chris Coyne <notifications@github.com> wrote:

> In shared/actions/tracker.js
> <https://github.com/keybase/client/pull/3187#discussion_r68391309>:
>
> > @@ -621,6 +631,26 @@ function getTracking (username: string): Promise {
> >    })
> >  }
> >
> > +function fillFolders (getState: () => {favorite: FavoriteState}, username: string): TrackerActionCreator {
> > +  return (dispatch, getState) => {
> > +    const root = getState().favorite.folders || {}
> > +    const pubIg = root.public.ignored || []
> > +    const pubTlf = root.public.tlfs || []
> > +    const privIg = root.private.ignored || []
> > +    const privTlf = root.private.tlfs || []
> > +
> > +    const tlfs = [].concat(pubIg, pubTlf, privIg, privTlf).filter(f => f.users.filter(u => u.username === username).length)
>
> I think I need some more context here, if you're asking me a q - not sure
> what I'm looking at. Let's talk in person.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3187/files/18bdb78f6f2f2c26e2a7609d0fc46f83fd641170#r68391309>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAfhUzLmu1mfolgOYLOAaCeQShvHrG2zks5qO86HgaJpZM4I9TZw>
> .
>